### DISCUSSION TITLE
Path filtering inside pull requests

### DISCUSSION THREAD

**Discussion by alexrosenfeld10** [community/community#27006] [Actions]
<p>Is it possible to filter file changes inside the “on: pull_request” event? Something like this:</p>
<pre><code>on:
  push:
    branches:
      - master
    paths:
      - 'thing/*'
  pull_request:
    branches:
      - master
    paths:
      - 'thing/*'
</code></pre>
<p>It doesn’t seem to work for me, the action runs even if the change is not in the <code>thing</code> directory </p>

**Comment by alexrosenfeld10** [comment, marked as answer]
<p>This is just me being dumb. It’s because <em>previous</em> commits in the PR had diffs in the file glob that the action was still being triggered. Duhh…</p>

**Reply by egithinji** [reply]
I was making the same mistake and your question and answer helped me!
